src/MongoDB.Analyzer.Package/MongoDB.Analyzer.Package.csproj

```diff
@@ -10,14 +10,14 @@
 
   <PropertyGroup>
     <PackageId>MongoDB.Analyzer</PackageId>
-    <PackageVersion>1.3.0</PackageVersion>
+    <PackageVersion>$(Version)</PackageVersion>
```

Comment:
Do we need this here? As far as I understood the package version is set by dotnet cli parameters.

Reply:
Leaving this for better readability (value in `PackageVersion` overrides the passed in cli)